src/datasets/utils/file_utils.py

```diff
@@ -386,6 +387,13 @@ def get_datasets_user_agent(user_agent: Optional[Union[str, dict]] = None) -> st
     return ua
 
 
+def ftp_get(url, temp_file, proxies=None, resume_size=0, user_agent=None, cookies=None):
+    logger.info(f"Getting through FTP {url} into {temp_file.name}")
+    print(f"Getting through FTP {url} into {temp_file.name}")
+    with closing(urllib.request.urlopen(url)) as r:
```

Comment:
same here

Reply:
Done. ftp_head returns a boolean whereas  ftp_get raises an Exception, I'm trying to follow the common code with Http as much as possible which I think uses the same path.